ad-joining/register-computer/main.py

```diff
@@ -255,8 +289,8 @@ def __register_computer(request):
         else:
             # There is an OU. That means the request is fine and we also know which
             # OU to create a computer account in.
-            computer_ou = matches[0].get_dn()
-            logging.info("Found OU '%s' to create computer account in, authorized (1/2)" % computer_ou)
+            project_ou = matches[0].get_dn()
```

Comment:
It is a little awkward that users have to create these project-OUs even if they use the custom OU features -- and I suppose there is a risk that AD admin comes along and deletes these OUs since they are empty and look superfluous. But I do not have a better idea right now...

Reply:
I agree. But we need this as a way for authorizing access from a VM to AD. A different design might be to add the invoker role and require users to add all related service accounts to the invokers list. This will be a breaking change for people using the GCF version of this service, and will lock this solution to Cloud Run (we won't be able to secure it the same way when the container is hosted on GKE)